arviz/rcparams.py

```diff
@@ -108,6 +108,7 @@ def _validate_boolean(value):
     "plot.bokeh.figure.height": (500, _validate_positive_int),
     "plot.bokeh.show": (True, _validate_boolean),
     "plot.matplotlib.constrained_layout": (True, _validate_boolean),
+    "plot.matplotlib.show": (True, _validate_boolean),
```

Comment:
I think I prefer false as default for matplotlib, because it is "backward compatible" and showing plots is easier than with bokeh, so calling `plt.show` from ArviZ is not that important.